.github/workflows/build.yml

```diff
@@ -353,18 +412,18 @@ jobs:
         fail-on-cache-miss: true
     - name: Setup environment variables
       run: |
-        PKG_CONFIG_PATH="$PWD/openssl1/build/lib/pkgconfig:$PWD/openssl3/build/lib64/pkgconfig:$PWD/wolfssl/build/lib/pkgconfig:$PWD/nghttp3/build/lib/pkgconfig"
-        LDFLAGS="-Wl,-rpath,$PWD/openssl1/build/lib -Wl,-rpath,$PWD/openssl3/build/lib64"
+        PKG_CONFIG_PATH="$PWD/openssl1/build/lib/pkgconfig:$PWD/openssl3/build/lib64/pkgconfig:$PWD/wolfssl/build/lib/pkgconfig:$PWD/nghttp3/build/lib/pkgconfig:$PWD/libressl/build/lib/pkgconfig"
+        LDFLAGS="-Wl,-rpath,$PWD/openssl1/build/lib -Wl,-rpath,$PWD/openssl3/build/lib64 -Wl,-rpath,$PWD/libressl/build/lib"
 
-        if [ "${{ matrix.tls }}" = "tls-reg" ]; then
+        if [ "${{ matrix.tls }}" = "group-b" ]; then
           PICOTLS_PREFIX="$PWD/picotls-openssl1"
-        else
+        elif [ "${{ matrix.tls }}" = "group-a" ]; then
           PICOTLS_PREFIX="$PWD/picotls-openssl3"
+        else
+          PICOTLS_PREFIX="$PWD/picotls-libressl"
         fi
-
         PICOTLS_CFLAGS="-I$PICOTLS_PREFIX/include/"
         PICOTLS_LIBS="-L$PICOTLS_PREFIX/build -lpicotls-openssl -lpicotls-core"
-
```

Comment:
Please do not remove empty lines unless it is absolutely needed for this change.